msal4j-brokers/src/main/java/com/microsoft/aad/msal4jbrokers/MsalRuntimeBroker.java

```diff
@@ -176,4 +176,37 @@ public boolean isBrokerAvailable() {
             return false;
         }
     }
+
```

Comment:
MSAL Java already has logging APIs that control both verbosity and PII. Why aren't those used ? We should not have special APIs for WAM logging.

Reply:
MSAL Java doesn't really have an API to control the verbosity: we send logs through a logging framework called SLF4J and we set the verbosity for those logs (info/debug/error/etc.) but it's up to an app developer to get a dependency to receive the logs for SLF4J, and through that separate dependency configure what level they want to see. 

Because we'd need a new method somewhere anyway, and since MSALRuntime logs are very verbose at every level (compare to MSAL Java at least), I figured it'd be more convenient to let app developers turn broker logs on/off as they need to (instead of introducing a new flag in MSAL Java itself).

As for PII, we allow app developers to turn it on at the app level, but there's no way to turn it off without completely recreating the PublicClientApplication object. This one could be easily hooked into MSAL Java's existing flag, but again since the MSALRuntime logs are created separately from MSAL Java and their available APIs behave differently, it made more sense to match the behavior rather than hide it.